lib/cli/src/commands/create_exe.rs

```diff
@@ -419,32 +418,102 @@ impl CreateExe {
                         let _ = std::fs::create_dir_all(&target_file_path);
                         let files = untar(local_tarball.clone(), target_file_path.clone())?;
                         tarball_dir = target_file_path.canonicalize().unwrap_or(target_file_path);
-                        files.iter().find(|f| f.contains(libwasmer_path)).cloned().ok_or_else(|| {
-                            anyhow!("Could not find libwasmer for {} target in the provided tarball path (files = {files:#?}, libwasmer_path = {libwasmer_path:?})", target)})?
+                        files.iter().find(|f| f.ends_with("libwasmer.a")).cloned().ok_or_else(|| {
```

Comment:
Can this change depending on OS? (e.g. we get `wasmer.dll` on Windows, or whatever)

Reply:
No, it's the same for all operating systems.